Twig/Lexer.php

```diff
@@ -2,39 +2,36 @@
 
 namespace PrestaShop\TranslationToolsBundle\Twig;
 
-class Lexer extends \Twig_Lexer
+use Twig\Source;
+use Twig\TokenStream;
+
+class Lexer extends \Twig\Lexer
 {
     /**
      * @var array
      */
     protected $comments = [];
 
-    /**
-     * @var bool
-     */
-    protected $call = false;
-
-    protected function lexComment()
+    public function tokenize($code, $name = null): TokenStream
     {
-        parent::lexComment();
-
-        if (true === $this->call) {
-            return;
+        if (!$code instanceof Source) {
+            $source = new Source($code, $name);
+        } else {
+            $source = $code;
```

Comment:
```
$source = $code;
$code = $source->getCode();
```
So somehow `$code = $code->getCode();` 😝  ?

Reply:
Yes, I've done it on purpose, I know it's a bit weird but that's kinda how twig itself handle it to be retrocompatible (https://github.com/twigphp/Twig/blob/1.x/src/Lexer.php#L165-L183)